substrate/frame/revive/src/tests/sol/contract.rs

```diff
@@ -210,6 +210,7 @@ fn call_revert() {
 }
 
 #[test]
+#[ignore] // TODO: ignore until we decide what is the correct way to handle this
```

Comment:
```suggestion
#[ignore = "TODO: ignore until we decide what is the correct way to handle this"]
```

Reply:
Why is it ignored again? FWIW this could just not execute in PVM for now if that makes it easier to test.